packages/http-client/src/ceramic-http-client.ts

```diff
@@ -176,15 +177,7 @@ export default class CeramicClient implements CeramicApi {
   }
 
   addDoctypeHandler<T extends Doctype>(doctypeHandler: DoctypeHandler<T>): void {
```

Comment:
This function doesn't seem to actually be called anywhere in our codebase.  If this is just for developers to use to load their custom doctypes, can we rename it and change it to take the DocType directly instead of the DoctypeHandler?

Reply:
Exactly. Well that would break the symmetry between core and http-client. In core the handler is needed because updates need to use the handler to apply records.

Reply:
ah right, makes sense